test/e2e/lib/flows/jetpack-connect-flow.js

```diff
@@ -19,9 +19,10 @@ import * as driverManager from '../driver-manager';
 import * as driverHelper from '../driver-helper';
 import * as dataHelper from '../data-helper';
 import JetpackConnectPage from '../pages/jetpack/jetpack-connect-page';
-import JetpackConnectSiteTypePage from '../pages/jetpack/jetpack-connect-site-type-page';
-import JetpackConnectSiteTopicPage from '../pages/jetpack/jetpack-connect-site-topic-page';
-import JetpackConnectUserTypePage from '../pages/jetpack/jetpack-connect-user-type-page';
+// These flows are disabled via `jetpack/connect/site-questions` feature flag in Calypso:
+// import JetpackConnectSiteTypePage from '../pages/jetpack/jetpack-connect-site-type-page';
+// import JetpackConnectSiteTopicPage from '../pages/jetpack/jetpack-connect-site-topic-page';
+// import JetpackConnectUserTypePage from '../pages/jetpack/jetpack-connect-user-type-page';
```

Comment:
"Marking as ignored" vs just code-commenting tests in this flow-file was discussed in p1562574334001800-slack-e2e-testing-discuss and conclusion was to hide them because I didn't just want to skip steps, but whole sections in flows.